webpack.config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
   },
   module: {
     preLoaders: [
-      { test: /\.tsx?$/, loader: 'tslint', include: path.join(__dirname, 'src') }
```

Comment:
This is just to remove the annoying linter. Visual studio code hates us enough for bad code :)